includes/sanitizers/class-amp-style-sanitizer.php

```diff
@@ -311,6 +311,11 @@ private function get_used_tag_names() {
 				$used_tag_names[ $el->tagName ] = true;
 			}
 			$this->used_tag_names = array_keys( $used_tag_names );
+			foreach ( $this->selector_mappings as $html_tag => $amp_tags ) {
+				if ( 0 < count( array_intersect( $this->used_tag_names, $amp_tags ) ) ) {
+					$this->used_tag_names[] = $html_tag;
```

Comment:
Note that `selector_mappings` does not consist of tag names but rather selector terms, for example `.pb_feed`.